marconi/src/Marconi/Indexers.hs

```diff
@@ -140,17 +144,50 @@ datumWorker Coordinator{_barrier} ch path = Datum.open path (Datum.Depth 2160) >
               slot   <- chainPointToSlotNo cp
               offset <- findIndex (any (\(s, _) -> s < slot)) events
               Ix.rewind offset index
+-- | does the transaction contain a targetAddress
+isInTargetTxOut
+    :: TargetAddresses              -- ^ non empty list of target address
+    -> C.TxOut C.CtxTx era    -- ^  a cardano transaction out that contains an address
+    -> Bool
+isInTargetTxOut targetAddresses (C.TxOut address _ _ _) = case  address of
+    (C.AddressInEra  (C.ShelleyAddressInEra _) addr) -> addr `elem` targetAddresses
+    _                                                -> False
+
+utxoHotStoreWorker :: QSemN -> TargetAddresses -> Worker
+utxoHotStoreWorker qsem targetAddresses Coordinator{_barrier} ch path =
```

Comment:
Didn't we get rid of the hot store?

Reply:
we did. I neglected to change the name. The utxoWorkers are slightly different. The new one needs to be aware of when queries are performed against the SQLite to avoid locking the database. 
I'll change the name to queryAwareUtxoWorker